<launch>

  <arg name="robot_namespace" default="/"/>
  <arg name="paused" default="false"/>
  <arg name="use_sim_time" default="false"/>
  <arg name="gui" default="true"/>
  <arg name="headless" default="false"/>
  <arg name="debug" default="false"/>

  <include file="$(find gazebo_ros)/launch/empty_world.launch"/>
  <param name="/real_time_factor" value="0.5" />

  <param name="robot_description" command="$(find xacro)/xacro '$(find new_quadruped_model_kp)/urdf/new_quadruped_model_kp.urdf.xacro'
    --inorder
    robot_namespace:=$(arg robot_namespace)" />

  <node
    name="spawn_quadruped_model"
    pkg="gazebo_ros"
    type="spawn_model"
    args="-urdf
    -urdf
    -model $(arg robot_namespace)
    -param robot_description
    -z 1.2"
   />

   <rosparam file="$(find new_quadruped_model_kp)/config/two_joint_controller.yaml" command="load"/>

   <!-- load the controllers -->
   <!-- <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
 	output="screen" args="single_joint_controller"/> -->

</launch>
